_benchmark/commands/execute-test.md

```diff
@@ -0,0 +1,178 @@
+| 90.0th percentile service time |     painless_dynamic |   407.579 |     ms |
+| 99.0th percentile service time |     painless_dynamic |   430.806 |     ms |
+| 99.9th percentile service time |     painless_dynamic |   457.352 |     ms |
+|  100th percentile service time |     painless_dynamic |   459.474 |     ms |
+
+----------------------------------
+[INFO] SUCCESS (took 2634 seconds)
+----------------------------------
+```
+
+## Options
+
+Use the following options to customize the `execute-test` command for your use case. Options in this section are categorized by their use-case.
+
+## General settings
+
+The following options shape how each test runs and how results are output.
+
+- `--test-mode`: Runs the given workload in test mode, which is useful when checking a workload for errors.
+- `--user-tag`: Defines user-specific key-value pairs to be used in metric record as meta information, for example, `intention:baseline-ticket-12345`.
+- `--results-format`: Defines the output format for the command line results, either `markdown` or `csv`. Default is `markdown`.
+- `--results-number-align`: Defines the column number alignment for when the `compare` command outputs results. Default is `right`.
+- `--results-file`: When provided a file path, writes the compare results to the file indicated.
+- `--show-in-results`: Determines whether or not to include the comparison in the results file.
+
+
+### Distributions
+
+The following options set which version OpenSearch and OpenSearch plugins the benchmark test uses.
+
+- `--distribution-version`: Downloads the specified OpenSearch distribution based on version number. For a list of released OpenSearch versions, see [Version history](https://opensearch.org/docs/version-history/).
+- `--distribution-repository`: Defines the repository from where the OpenSearch distribution should be downloaded. Default is `release`.
+- `--revision`: Defines the current source code revision for which to run a benchmark test. Default is `current`.
+   - `current`: Uses the source tree's current revision based on your OpenSearch distribution. 
+   - `latest`: Fetches the latest revision from the main branch of the source tree.
+   - You can also a time stamp or commit ID from the source tree. When using a timestamp, specify `@ts` where "ts" is a valid ISO 8601 timestamp, for example `@2013-07-27T10:37:00Z`.
+-  `--opensearch-plugins`: Defines which [OpenSearch plugins]({{site.url}}{{site.baseurl}}/install-and-configure/plugins/) to install. By default, no plugins are installed.
+- `--plugin-params:` Defines a comma-separated list of key:value pairs that are injected verbatim into all plugins as variables.
+- `--runtime-jdk`: The major version of JDK to use.
+- `--client-options`: Defines a comma-separated lists of clients to use. All options are passed to the OpenSearch Python client. Default is `timeout:60`.
+
+### Cluster
+
+The following options relate to the target cluster of the benchmark.
```

Comment:
Is the intention that more options will be added at some time in the future? You may still want to be accurate and say "following options relates to ..."